XMonad/Layout/Minimize.hs

```diff
@@ -112,7 +112,7 @@ instance LayoutModifier Minimize Window where
                 case M.lookup w (W.floating ws) of
                   Nothing -> return $ Just $ Minimize (w:minimized) unfloated
                   Just r -> do
-                    modify (\s -> s { windowset = W.sink w ws})
+                    windows (W.sink w)
```

Comment:
Oh, wtf is going on? This has obviously nothing to do with the Spacing thing. Why is this even in this pull request?
I found a minor bug with Minimize that this commit fixes, but I realized that Minimize prior used `windows` instead of `modify` and changed it because it caused problems with decorations for exactly the mentioned reasons. I didn't have the time to make this properly, so I just left it there.
How do I fix this? Help!